### PR TITLE
Bugfix happ-642

### PR DESCRIPTION
**Pull request by amit-metri**
-bugfix happ-642

### CODE REVIEW

File: repository/src/main/java/ca/bc/gov/repository/bcsc/BcscAuthRepo.kt [hunk 1]
```diff
@@ -104,9 +113,7 @@ class BcscAuthRepo(
     * Check for logged in session
     * */
     suspend fun checkLogin(): Boolean {
```
Comment (pinakinkansara) [inline repository/src/main/java/ca/bc/gov/repository/bcsc/BcscAuthRepo.kt:115]: rename this method to isSessionActive or checkSession

File: app/src/main/java/ca/bc/gov/bchealth/ui/healthrecord/individual/IndividualHealthRecordFragment.kt [hunk 2]
```diff
@@ -257,19 +260,22 @@ class IndividualHealthRecordFragment : Fragment(R.layout.fragment_individual_hea
         }
     }
 
-    private fun updateUi(uiState: IndividualHealthRecordsUiState) {
+    private suspend fun updateUi(uiState: IndividualHealthRecordsUiState) {
```
Comment (pinakinkansara) [inline app/src/main/java/ca/bc/gov/bchealth/ui/healthrecord/individual/IndividualHealthRecordFragment.kt:263]: remove suspend modifier from this method

**Review by pinakinkansara — CHANGES_REQUESTED** [review]

File: repository/src/main/java/ca/bc/gov/repository/bcsc/BcscAuthRepo.kt (hunk 1 above)
Comment (amit-metri) [reply repository/src/main/java/ca/bc/gov/repository/bcsc/BcscAuthRepo.kt]: Done

File: app/src/main/java/ca/bc/gov/bchealth/ui/healthrecord/individual/IndividualHealthRecordFragment.kt (hunk 2 above)
Comment (amit-metri) [reply app/src/main/java/ca/bc/gov/bchealth/ui/healthrecord/individual/IndividualHealthRecordFragment.kt]: removed suspend function in the view layer

**Review by pinakinkansara — APPROVED** [review]

**Pull request merged**